src/rs485/protocols/hippofeed.cpp

```diff
@@ -0,0 +1,25 @@
+#include "rs485/protocols/hippofeed.h"
+#include "rs485/protocols/checksums/modbus_rtu.h"
+#include <gtest/gtest.h>
+
+PacketStatus HippoFeedProtocol::isPacket(const RS485BusBase& bus, size_t startIndex, size_t& endIndex) const {
+  ModbusRTUChecksum checksum;
+
+  if(startIndex == endIndex) return PacketStatus::NOT_ENOUGH_BYTES;  // Cannot see the length byte
```

Comment:
It seems to me that this should make sure there are *at least* four bytes in the buffer.